app/v2/notifications/post_notifications.py

```diff
@@ -302,7 +302,9 @@ def save_email_or_sms_to_queue(
     encoded = signing.encode(data)
```

Comment:
just realised we could write something like:
```suggestion
    # include template_has_unsubscribe_link in the call to the task but dont try and return it from the function in the Notification object
    encoded = signing.encode(data | {'"template_has_unsubscribe_link": template.has_unsubscribe_link})
```